phonebook/main_test.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestCreatePhoneBook(t *testing.T) {
+	tests := []struct {
+		name           string
+		names, numbers []string
+		want           map[string]string
+	}{
+		{
+			name:    "empty",
+			names:   []string{},
+			numbers: []string{},
+			want:    map[string]string{},
+		},
+		{
+			name:    "single letter",
+			names:   []string{"a"},
+			numbers: []string{"0"},
+			want:    map[string]string{"a": "0"},
+		},
+		{
+			name:    "no name, only phone number",
```

Comment:
nit: the description is not quite correct. There is a name, it's just an empty string.